robot_ipsa:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  arm1_position_controller:
    type: effort_controllers/JointPositionController
    joint: base_link_to_arm1
    pid: {p: 350.0, i: 0.35, d: 35.0}
  arm2_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm1_to_arm2
    pid: {p: 250.0, i: 0.25, d: 25.0}
  arm3_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm2_to_arm3
    pid: {p: 30.0, i: 0.03, d: 3.0}
